### ISSUE TITLE
Problems with creating new REPL

### ISSUE THREAD

**Issue by kinnala** [dzop/emacs-jupyter#54]
I'm on master and Emacs 26.1.

I can successfully connect to existing jupyter kernel with `jupyter-connect-repl`. The REPL seems to work fine. All commands and plotting works as I'd expect.

However, I cannot start a new REPL using `jupyter-run-repl`. I get the following error
```
jupyter-read-kernelspec: No kernel.json file found in /home/tom/src/venv/my-venv-name/lib/python3.5/site-packages/ipykernel/resources
```
Any tips for solving this? I have also problems with the org integration but I think it might be related to this.

**Comment by dzop** [comment]
It looks like there is something wrong with how your [kernelspec](https://jupyter-client.readthedocs.io/en/stable/kernels.html#kernelspecs) was setup. `emacs-jupyter` looks at the output of the shell command `jupyter kernelspec list` to get the directories of the kernelspecs and those directories are assumed to contain a `kernel.json` file.

Can you maybe try to re-install the kernelspec using the advice in https://github.com/dzop/emacs-jupyter/issues/23#issuecomment-462072769.

**Comment by kinnala** [comment]
Thanks! I wrote `jupyter kernelspec list` which pointed to a directory `~/src/venv/my-venv-name/lib/python3.5/site-packages/ipykernel/resources/` which had no `kernel.json`. I'm running quite old versions of Python et co so it might also be the reason for that but not sure.

I ran the command in your link:
```
python -m ipykernel install --user --name ENVNAME --display-name "Python (whatever you want to call it)"
```
which created a new entry to `jupyter kernelspec list`. This was not enough because `jupyter-run-repl` still tried to use the first entry in `jupyter kernelspec list` which pointed to this empty directory. So I manually copied `kernel.json` from the new directory to the empty directory and it started working.
